lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -374,6 +357,30 @@ export class NativeInteractionClient extends BaseInteractionClient {
         return result;
     }
 
+    protected addTelemetryFromNativeResponse(response: NativeResponse): MATS | null {
+
+        const mats = this.getMATSFromResponse(response);
+        this.performanceClient.addStaticFields({
+            extensionId: this.nativeMessageHandler.getExtensionId(),
+            extensionVersion: this.nativeMessageHandler.getExtensionVersion(),
+            matsBrokerVersion: mats ? mats.broker_version : undefined,
```

Comment:
I know this code was simply moved from somewhere else but can we do an early return if `mats` is undefined instead of doing `mats ? mats.param : undefined` everywhere?

Reply:
Do we just want to return null or do we want to return performanceClient with static fields that don't need mats?

Reply:
What did we decide here?

Reply:
@sameerag  we are returning null if mats is null